src/client/java/teammates/client/scripts/sql/VerifyNonCourseEntityAttributesBaseScript.java

```diff
@@ -128,12 +133,24 @@ protected List<Map.Entry<T, E>> checkAllEntitiesForFailures() {
             log(String.format("Verification Progress %d %%",
                      (int) ((float) currPageNum / (float) numPages * 100)));
 
+            long startTimeForSql = System.currentTimeMillis();
             List<T> sqlEntities = lookupSqlEntitiesByPageNumber(currPageNum);
+            long endTimeForSql = System.currentTimeMillis();
+            log("Querying for SQL for page " + currPageNum + " took "
```

Comment:
Do these timings (here and `DataMigrationEntitiesBaseScriptSql.java`) affect how readable the logged output is? If it is perhaps we can remove it from the PR or add a flag/ log option to only output on debug or something

Reply:
It's less verbose because I've adjusted batch size to be 1000 instead of 100. The log statements can also show us what to further optimize if it's still slow during migration.